PORTSCAN

```diff
@@ -176,6 +187,7 @@ DO FOREVER
   if excp_sock \= "" then do
     if debug then call OutDebug "EXCEPTION SOCKETS:" excp_sock
     Do i = 1 to WORDS(excp_sock)
+     read_sockets.0 = read_sockets.0 + 1
```

Comment:
Why is this here?

Reply:
Wrong copy paste. I updated my repo, new PR incoming soon.